src/features/search/components/SearchFilterButton/SearchFilterButton.native.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { SearchFilterButton } from 'features/search/components/SearchFilterButton/SearchFilterButton'
+import { render } from 'tests/utils'
+import { Check } from 'ui/svg/icons/Check'
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Si on a mis une règle ce n'est pas pour l'autorisé, si c 'est impossible autrement why not, mais la on peut importer le theme `import { theme } from 'theme'`